code/datums/datumvars.dm

```diff
@@ -1176,7 +1176,7 @@
 		possibleverbs += typesof(/mob/proc,/mob/verb,/mob/living/proc,/mob/living/verb)
 		switch(H.type)
 			if(/mob/living/carbon/human)
-				possibleverbs += typesof(/mob/living/carbon/proc,/mob/living/carbon/verb,/mob/living/carbon/human/verb,/mob/living/carbon/human/proc)
+				possibleverbs += typesof(/mob/living/carbon/proc,/mob/living/carbon/human/verb,/mob/living/carbon/human/proc)
```

Comment:
Why did you change this?

Reply:
So it turns out that was only one carbon verb, that being `give`. Removing it as a verb made this throw a compilation error.